python/tk_framework_desktopserver/server_protocol.py

```diff
@@ -26,6 +23,8 @@
 from autobahn.twisted.websocket import WebSocketServerProtocol
 from twisted.internet import error, reactor
 
+logger = get_logger(__name__)
```

Comment:
I did this a while back @thebeeland , seems like I'll be stepping on your toes a bit.